plugins/dialog/guest-js/index.ts

```diff
@@ -30,7 +30,14 @@ interface DialogFilter {
 interface OpenDialogOptions {
   /** The title of the dialog window (desktop only). */
   title?: string
-  /** The filters of the dialog. */
+  /**
+   * The filters of the dialog.
+   * The behavior of filters on mobile platforms (iOS and Android) is different from desktop.
+   * On Android, we are not able to filter by extension, only by MIME type.
+   * On iOS, we are able to filter by extension in the document picker, but not in the media picker.
+   * In practice, if an extension for a video or image is included in the filters (such as `png` or `mp4`),
+   * the media picker will be displayed instead of the document picker (regardless of the {@linkcode pickerMode} value).
```

Comment:
is this a ios limitation or our implementation (i didn't exactly study the code yet but i saw the filtersIncludeImage).

if this is ios' fault then alright, but if not, what i want to filter for jpg and pdf at the same time for example?